### PR TITLE
abcde: Fixed musicbrainz backend

### PR DESCRIPTION
**Pull request by faulesocke**
These packages are all required to make abcde work with the default configuration.

Another option is to not provide these at all and change the default config to not use musicbrainz. I asked in the IRC what would be the best thing to do and was told I should package the additional perl packages.

### CODE REVIEW

File: srcpkgs/perl-Class-Accessor/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'perl-Class-Accessor'
+pkgname=perl-Class-Accessor
+_dist=Class-Accessor
```
Comment (maxice8) [inline srcpkgs/perl-Class-Accessor/template]: uncessary, just do ${pkgname//perl/}.

File: srcpkgs/perl-Class-Accessor/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+version=0.51
+revision=1
+wrksrc=$_dist-$version
```
Comment (maxice8) [inline srcpkgs/perl-Class-Accessor/template]: double quotes required.

File: srcpkgs/perl-Class-Accessor/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+build_style=perl-module
+hostmakedepends="perl"
+depends="perl"
+short_desc="Automated accessor generation"
+maintainer="Urs Schulz <voidpkgs@ursschulz.de>"
+license="GPL-1 PerlArtistic"
```
Comment (maxice8) [inline srcpkgs/perl-Class-Accessor/template]: Artistic-1.0-Perl, GPL-1.0-or-later

File: srcpkgs/perl-Class-Accessor/template [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+homepage="http://search.cpan.org/dist/$_dist/"
+distfiles="$CPAN_SITE/Class/$_dist-$version.tar.gz"
```
Comment (maxice8) [inline srcpkgs/perl-Class-Accessor/template]: ditto

File: srcpkgs/perl-Mojolicious/template [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'perl-Mojolicious'
+pkgname=perl-Mojolicious
+_dist=Mojolicious
```
Comment (maxice8) [inline srcpkgs/perl-Mojolicious/template]: See the comments on perl-Class-Accessor, make all changes that also apply.

File: srcpkgs/perl-MusicBrainz-DiscID/template [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'perl-musicbrainz-discid'
+pkgname=perl-MusicBrainz-DiscID
+_dist="MusicBrainz-DiscID"
```
Comment (maxice8) [inline srcpkgs/perl-MusicBrainz-DiscID/template]: See the comments on perl-Class-Accessor, make all changes that also apply.

File: srcpkgs/perl-WebService-MusicBrainz/template [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'perl-WebService-MusicBrainz'
+pkgname=perl-WebService-MusicBrainz
+_dist=WebService-MusicBrainz
```
Comment (maxice8) [inline srcpkgs/perl-WebService-MusicBrainz/template]: See the comments on perl-Class-Accessor, make all changes that also apply.

**Comment by faulesocke** [comment]
Now they are nice and shiny.

**Pull request merged**